includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -369,18 +372,56 @@ protected function get_row_data( $item ) {
 					$course_title = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . apply_filters( 'the_title', $item->post_title, $item->ID ) . '</a></strong>';
 				}
 
-				$column_data = apply_filters(
-					'sensei_analysis_overview_column_data',
-					array(
-						'title'              => $course_title,
-						'last_activity'      => $last_activity_date,
-						'completions'        => $course_completions,
-						'average_percent'    => $average_grade,
-						'days_to_completion' => $average_completion_days,
-					),
-					$item,
-					$this
+				// Fetch learners in course.
+				$course_args           = array(
+					'post_id' => $item->ID,
+					'type'    => 'sensei_course_status',
+					'status'  => 'any',
 				);
+				$course_students_count = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_learners', $course_args, $item ) );
+
+				// Calculate average lesson progress per student for course.
+				$average_course_progress = 0;
+
+				// Get all course lessons.
+				$lessons        = Sensei()->course->course_lessons( $item->ID, 'any', 'ids' );
+				$course_lessons = is_array( $lessons ) ? $lessons : array( $lessons );
+				$total_lessons  = count( $course_lessons );
+
+				// Get all completed lessons.
+				$completed_count        = 0;
+					$lesson_args        = array(
+						'post__in' => $course_lessons,
+						'type'     => 'sensei_lesson_status',
+						'status'   => array( 'graded', 'ungraded', 'passed', 'failed', 'complete' ),
+						'count'    => true,
+					);
+					$lesson_completions = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_lesson_completions', $lesson_args, $item ) );
+					if ( $lesson_completions ) {
+						$completed_count += (int) $lesson_completions;
```

Comment:
Do we need the '+=' here?

Reply:
oh nooo it waas before when I had for loop here, forgot to remove it. Thanks :D

Reply:
Removed [everything](https://github.com/Automattic/sensei/pull/4917/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR398)